src/utils/imageCache.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs/promises'
```

Comment:
Should we move this file to `src/modules/CourseThumbnailAPI/utils`?

Reply:
Initially I thought about that too, but leaving it here in case someone else might use it